src/common/ArticleCard.ts

```diff
@@ -0,0 +1,27 @@
+export interface Tag {
+  title: string
+}
+
+export interface FileDetails {
+  type: string
+  size: string
+}
+
+interface Image {
+  src: string
+  alt: string
+}
+
+export interface ArticleCardProps {
+  type: "articleCard"
+  lastUpdated: string
+  category: string
+  title: string
+  url: string
+  description: string
+  image?: Image
+  variant: "file" | "article"
+  fileDetails?: FileDetails
```

Comment:
supernit: can we make this prop only available if the variant is set to `file`? That way it can be set as required.

Reply:
[79659e4](https://github.com/isomerpages/isomer-components/pull/45/commits/79659e486148c1f71e324b29c01eb7896623372d)